cnf-tests/.konflux/Dockerfile

```diff
@@ -0,0 +1,74 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-ose-cli-rhel9:v4.19 AS oc
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.2 AS builder-stresser
+ENV PKG_NAME=github.com/openshift-kni/cnf-features-deploy
+ENV PKG_PATH=/go/src/$PKG_NAME
+ENV TESTER_PATH=$PKG_PATH/cnf-tests/pod-utils/stresser
+RUN mkdir -p $PKG_PATH
+COPY . $PKG_PATH/
+WORKDIR $TESTER_PATH
+RUN go build -mod=vendor -o /stresser
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.2 AS builder-sctptester
+ENV PKG_NAME=github.com/openshift-kni/cnf-features-deploy
+ENV PKG_PATH=/go/src/$PKG_NAME
+ENV TESTER_PATH=$PKG_PATH/cnf-tests/pod-utils/sctptester
+RUN mkdir -p $PKG_PATH
+COPY . $PKG_PATH/
+WORKDIR $TESTER_PATH
+RUN go build -mod=vendor -o /sctptest
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.2 AS builder-hugepages-allocator
+ENV PKG_NAME=github.com/openshift-kni/cnf-features-deploy
+ENV PKG_PATH=/go/src/$PKG_NAME
+ENV TESTER_PATH=$PKG_PATH/cnf-tests/pod-utils/hugepages-allocator
+RUN mkdir -p $PKG_PATH
+COPY . $PKG_PATH/
+WORKDIR $TESTER_PATH
+RUN go build -mod=vendor -o /hugepages-allocator
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.2 AS builder-latency-test-runners
+ENV PKG_NAME=github.com/openshift-kni/cnf-features-deploy
+ENV PKG_PATH=/go/src/$PKG_NAME
+ENV TESTER_PATH=$PKG_PATH/cnf-tests/pod-utils
+RUN mkdir -p $PKG_PATH
+COPY . $PKG_PATH/
+WORKDIR $TESTER_PATH
+RUN go build -mod=vendor -o /oslat-runner oslat-runner/main.go && \
+    go build -mod=vendor -o /cyclictest-runner cyclictest-runner/main.go && \
+    go build -mod=vendor -o /hwlatdetect-runner hwlatdetect-runner/main.go
+
+
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:v1.23.2 AS gobuilder
+WORKDIR /app
+COPY . .
+RUN make test-bin
+
+FROM registry.redhat.io/ubi9/ubi-minimal:9.5
```

Comment:
The container image should be based on the same RHEL version as OCP. Let's use a `RHEL_VERSION`  to be able to override it in case needed. Please see https://github.com/openshift-kni/dpdk-base/pull/5 for reference.

OCP 4.19 GA will be based on RHEL 9.4 EUS so we should use RHEL version as 9.4 here.

Reply:
Thanks for highlighting and for the reference, I'll adopt this.